src/about.html

```diff
@@ -0,0 +1,169 @@
+            </ul>
+          </div>
+        </nav>
+        <button class="navbar__cta-button" type="button">
+          Get Your Ticket
+        </button>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container">
+        <div class="headline__overtitle">"Hello! JavaScript Lovers"</div>
+        <h1 class="headline__title">
+          Global JavaScript Conference <br />
+          2022
+        </h1>
+        <p class="headline__description">
+          JavaScript is #1 on demand programming language. Meet the ones who
+          shape out the now and the future of the prospering nation.
+        </p>
+        <div class="headline__contact">
+          Please contact us per Email for any further questions about CC Global
+          Summit 2015! <br />
+          <a
+            class="headline__contact-link"
+            href="mailto:ccsummit2015@cckorea.org"
+            >ccsummit2015@cckorea.org</a
+          >
+        </div>
+      </div>
+    </section>
+    <section class="section">
+      <div class="container">
+        <h2 class="section__title">Past Conferences</h2>
+        <div class="section__subtitle">
+          Take a look at the last two JavaScript Global Conferences which took
+          place in Buenos Aires and in Warsaw.
+        </div>
+        <div class="section__body section__body--desktop-grid">
+          <div class="event-card">
+            <div class="event-card__year">2013</div>
+            <h2 class="event-card__title">
+              JS Global Conference, 2013 in Buenos Aires
+            </h2>
+          </div>
+          <div class="event-card">
+            <div class="event-card__year">2011</div>
+            <h2 class="event-card__title">
+              JS Global Conference, 2011 in Warsaw
+            </h2>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section section--dark">
+      <div class="container">
+        <h2 class="section__title">Partners</h2>
+        <div class="section__body">
+          <img
+            class="partner"
+            src="./assets/images/partner-1.png"
+            alt="Partner 1"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-2.png"
+            alt="Partner 2"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-3.png"
+            alt="Partner 3"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-4.png"
+            alt="Partner 4"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-5.png"
+            alt="Partner 5"
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      Event by
+      <img
+        height="40"
+        width="40"
+        src="./assets/images/git-nation.svg"
+        alt="Git Nation"
+      />
+      <strong>GitNation</strong>
```

Comment:
- [ ] Make sure to add a p tag and write something `@copy capstone 2022 `. Follow the attached loom video to understand better.